osi_vehicle.proto

```diff
@@ -0,0 +1,829 @@
+
+            // Request to an ADAS-Function for a lane change.
+            //
+            optional LaneChangeRequest lane_change_request = 8;
+
+            // Request to an ADAS-Function for a lane change.
+            //
+            enum LaneChangeRequest
+            {
+                // Stay on the actual lane.
+                //
+                LANE_CHANGE_REQUEST_EGO_LANE = 0;
+
+                // Change to the left.
+                //
+                LANE_CHANGE_REQUEST_LC_LEFT = 1;
+
+                // Change to the right.
+                //
+                LANE_CHANGE_REQUEST_LC_RIGHT = 2;
+            }
+        }
+    }
+}
+
+
+//
+// \brief A description for the steering wheel.
+//
+message SteeringWheel
+{
+    // Angle of the steering wheel. 
+    // 0=Central (Straight); Left>0; 0>Right.
+    //
+    // Unit: [rad]
+    //
+    optional double angle = 1;
+
+    // Angle-speed of the steering wheel.
+    // 0=Central (Straight); Left>0; 0>Right.
+    //
+    // Unit: [rad/s]
+    //
+    optional double angular_speed = 2;
+
+    // Torque of the steering wheel to the hand.
+    // 0=Central (Straight); Left>0; 0>Right.
+    //
+    // Unit: [N*m]
+    //
+    optional double torque = 3;
+}
+
+//
+// \brief A description for the positions of the pedals.
+//
+//
+message Pedalry
+{
+    // Position of the acceleration-pedal.
+    // Unit: [0-1] (Unpressed - fully pressed)
+    //
+    optional double pedal_position_acceleration = 1;
+
+    // Position of the brake-pedal.
+    // Unit: [0-1] (Unpressed - fully pressed)
+    //
+    optional double pedal_position_brake = 2;
+
+    // Position of the clutch-pedal.
+    // Unit: [0-1] (Unpressed - fully pressed)
+    //
+    optional double pedal_position_clutch = 3;
+}
+
+//
+// \brief This is a message to describe, which trajectory the vehicle should
+// follow.
+//
+//
+message Trajectory
+{
+    // Contains the timestamp where the trajectorypoint should be reached.
+    //
+    // Unit: [s]
+    //
+    optional Timestamp timestamp = 1;
+
+    // Contains the X-Position the vehicle should be at the timestamp.
+    //
+    optional double targeted_pos_x = 2;
+
+    // Contains the Y-Position the vehicle should be at the timestamp.
+    //
+    optional double targeted_pos_y = 3;
+
+    // Direction of the vehicle at the timestamp.
+    //
+    // Unit: [rad]
+    //
+    optional double track_angle = 4;
+
+    // Contains the curvature at the timestamp.
+    //
+    // Unit: [1/m]
+    //
+    optional double curvature = 5;
+
+    // Contains the curvature change at the timestamp.
+    //
+    // Unit: [1/(m*s)]
+    //
+    optional double curvature_change = 6;
+
+    // Contains the velocity of the vehicle at the timestamp.
+    //
+    // Unit: [m/s]
+    //
+    optional double velocity = 7;
+
+    // Contains the acceleration of the vehicle at the timestamp.
+    //
+    // Unit: [m/s^2]
+    //
+    optional double acceleration = 8;
+
```

Comment:
Isnt that redundant information to mention here accleration and velocity again? See VehicleKinematics --> Base Moving

Reply:
meant is the targeted velocity and acceleration at the planned point.. but I am not sure here